src/main.js

```diff
@@ -92,12 +96,31 @@ export class Draggable {
             const { which } = e;
 
             if ((which && which > 1) || this._ignoreMouse) {
+                bind(this.document, "contextmenu", preventDefault);
+                
+                this.cancelDrag(e);
+
+                this._cancelHandler(e);
```

Comment:
It seems like we better reverse the order and act depending on whether the `dragCancel` event is prevented or not - consider the scenario where consumers of the package are notified for the `dragCancel` event, but they prevent it - then the `dragCancel` event should allow not cancelling the `dragCancel` event, thus the `cancelDrag` function should not be executed.

The above is only true if we actually want to allow preventing events, which I don't think is currently supported in the `kendo-draggable` package. Still, it is still valid to think if reversing the order is better as we notify for the occurrence of the `dragCancel` and then we actually cancel it. It will also help if we introduce something like `dragCancelEventArgs.preventDefault` someday.